packages/expo-modules-core/ios/Swift/ModulesProvider.swift

```diff
@@ -18,6 +18,12 @@ public protocol ModulesProviderProtocol: ModulesProviderObjCProtocol {
    Returns an array of classes that hooks into `ExpoAppDelegate` to receive app delegate events.
    */
   func getAppDelegateSubscribers() -> [ExpoAppDelegateSubscriber.Type]
+
+  /**
+   Returns an array of (packageName, klass) tuple for `ReactDelegate` to hook React instance creation.
+   */
+  typealias ReactDelegateHandlerTupleType = (packageName: String, klass: EXReactDelegateHandler.Type)
+  func getReactDelegateHandlers() -> [ReactDelegateHandlerTupleType]
```

Comment:
This comment documents the alias, not the function 😉 Replace the order or add the comment for typealias too.